wearipedia/devices/garmin/fenix_gen.py

```diff
@@ -12,409 +13,2833 @@
+                {"id": "SAVE_LOCATION", "index": 4, "required": None},
+                {"id": "DO_NOT_DISTURB", "index": 5, "required": None},
+                {"id": "BLUETOOTH", "index": 6, "required": None},
+                {"id": "STOPWATCH", "index": 7, "required": None},
+                {"id": "BRIGHTNESS", "index": 8, "required": None},
+                {"id": "LOCK_DEVICE", "index": 9, "required": None},
+                {"id": "SYNC", "index": None, "required": None},
+                {"id": "SET_TIME", "index": None, "required": None},
+                {"id": "ALARMS", "index": None, "required": None},
+                {"id": "TIMER", "index": None, "required": None},
+                {"id": "FLASHLIGHT", "index": None, "required": None},
+            ],
+            "customUserText": None,
+            "metricsFileTrueupEnabled": True,
+            "relaxRemindersEnabled": True,
+            "smartNotificationTimeout": "MEDIUM",
+            "intensityMinutesCalcMethod": "AUTO",
+            "moderateIntensityMinutesHrZone": 3,
+            "vigorousIntensityMinutesHrZone": 4,
+            "keepUserNamePrivate": None,
+            "audioPromptLapEnabled": False,
+            "audioPromptSpeedPaceEnabled": False,
+            "audioPromptSpeedPaceType": "AVERAGE",
+            "audioPromptSpeedPaceFrequency": "INVALID",
+            "audioPromptSpeedPaceDuration": 180,
+            "audioPromptHeartRateEnabled": False,
+            "audioPromptHeartRateType": "HEART_RATE",
+            "audioPromptHeartRateFrequency": "INVALID",
+            "audioPromptHeartRateDuration": 180,
+            "audioPromptDialectType": None,
+            "audioPromptActivityAlertsEnabled": False,
+            "audioPromptPowerEnabled": False,
+            "audioPromptPowerType": "AVERAGE",
+            "audioPromptPowerFrequency": "INVALID",
+            "audioPromptPowerDuration": 180,
+            "weightOnlyModeEnabled": None,
+            "phoneNotificationPrivacyMode": "OFF",
+            "diveAlerts": None,
+            "liveEventSharingEnabled": None,
+            "liveTrackEnabled": None,
+            "liveEventSharingEndTimestamp": None,
+            "liveEventSharingMsgContents": None,
+            "liveEventSharingTargetDistance": None,
+            "liveEventSharingMsgTriggers": None,
+            "liveEventSharingTriggerDistance": None,
+            "liveEventSharingTriggerTime": None,
+            "dbDrivenDefaults": None,
+            "schoolMode": None,
+            "customMeasurementDate": None,
+            "customBodyFatPercent": None,
+            "customMuscleMass": None,
+            "customDeviceWeight": None,
+            "customDeviceBodyFatPercent": None,
+            "customDeviceMuscleMass": None,
+            "vivohubEnabled": None,
+        }
+
+        device_settings.append(device_data)
+
+    return device_settings
+
+
+def get_device_alarms_data(start_date, num_days):
```

Comment:
Your next three functions (up to line 2213) return empty lists. Is it necessary to have all these functions separately?

Reply:
@TrafficCop There doesn't exist any real data which I can use to generate synthetic data.